Tasks/ShellScript/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -8,10 +8,10 @@
   "loc.input.help.scriptPath": "Relative path from repo root of the shell script file to run.",
   "loc.input.label.args": "Arguments",
   "loc.input.help.args": "Arguments passed to the shell script",
-  "loc.input.label.cwd": "Working Directory",
-  "loc.input.help.cwd": "Current working directory when script is run.  Defaults to the folder where the script is located.",
-  "loc.input.label.disableAutoCwd": "Disable Auto CWD",
+  "loc.input.label.disableAutoCwd": "Specify Working Directory",
   "loc.input.help.disableAutoCwd": "The default behavior is to set the working directory to the script location. This enables you to optionally specify a different working directory.",
+  "loc.input.label.cwd": "Working Directory",
+  "loc.input.help.cwd": "Current working directory when script is run. Defaults to the $(System.DefaultWorkingDirectory).",
```

Comment:
I updated this description as well. The Working Directory input is only visible now when the Specify Working Directory checkbox is checked.